core/controllers/release_coordinator.py

```diff
@@ -49,3 +54,121 @@ def get(self) -> None:
     def delete(self) -> None:
         caching_services.flush_memory_caches()
         self.render_json({})
+
+
+class FeatureFlagsNormalizePayloadDict(TypedDict):
+    """Dict representation of FeatureFlag's normalized_payload
+    dictionary.
+    """
+
+    action: Optional[str]
```

Comment:
Why are these optional? Shouldn't they be required?

Reply:
I have updated this.